alpha/lib/model/EdgeServer.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+
+/**
+ * Skeleton subclass for representing a row from the 'edge_server' table.
+ *
+ * 
+ *
+ * You should add additional methods to this class to meet the
+ * application requirements.  This class will only be generated as
+ * long as it does not already exist in the output directory.
+ *
+ * @package Core
+ * @subpackage model
+ */
+class EdgeServer extends BaseEdgeServer {
+	
+	const CUSTOM_DATA_DELIVERY_IDS = 'delivery_profile_ids';
+
+	/**
```

Comment:
The Ctor was auto generated so I did not want to remove it to not create future diffs.